scripts/if_changed.sh

```diff
@@ -66,6 +66,10 @@ else
       check_changes '^(Firestore/(core|third_party)|cmake)'
       ;;
 
+    GoogleDataLogger-*)
+      check_changes '^(GoogleDataLogger|GoogleDataLogger.podspec)'
```

Comment:
The `pod lib lint` tests should be in the GoogleDataLogger travis job or another one just for the GoogleDataLogger pod lib lint test.